privacyidea/static/components/token/views/token.details.html

```diff
@@ -555,25 +558,53 @@ <h4 ng-show="token.user_id" translate>Assigned User</h4>
 
 
 <div ng-show="checkRight('assign')">
-<h4 ng-hide="token.user_id || token.locked" translate>
-    Assign User</h4>
-<form ng-hide="token.user_id || token.locked"
-      name="formAssignToken"
-      role="form" validate>
-    <div assign-user new-user-object="newUser" realms="realms"></div>
+    <h4 ng-hide="token.user_id || token.locked" translate>
+        Assign User</h4>
+    <form ng-hide="token.user_id || token.locked"
+          name="formAssignToken"
+          role="form" validate>
+        <div assign-user new-user-object="newUser" realms="realms" enable-set-pin=true></div>
 
-    <div class="text-center">
-        <button type="button" ng-click="assignUser()"
-                id="assignButton"
-                ng-disabled="formAssignToken.$invalid"
-                class="btn btn-primary">
-            <span class="glyphicon glyphicon-resize-small"></span>
-            <span translate>Assign User</span>
-        </button>
-    </div>
-</form>
+        <div class="text-center">
+            <button type="button" ng-click="assignUser()"
+                    id="assignButton"
+                    ng-disabled="formAssignToken.$invalid"
+                    class="btn btn-primary">
+                <span class="glyphicon glyphicon-resize-small"></span>
+                <span translate>Assign User</span>
+            </button>
+        </div>
+    </form>
 </div>
 
+<!-- ============ Container ====================== -->
+<h4>Assign Container</h4>
```

Comment:
So if i understand this correctly, we don't have a "container_read" right and thus the "Assign Container" block will always be displayed in a token view (and other views)?
Also the heading should be translatable.